test/verify/check-superuser

```diff
@@ -245,6 +261,23 @@ class TestSuperuserOldShell(MachineCase):
         "machine2": {"address": "10.111.113.2/20", "image": "centos-7"},
     }
 
+    def checkSuperuserIndicator(self, access):
```

Comment:
These two helpers seem common enough to move to the `Browser` class in testlib.py, as they would often even apply to third-party projects. Even now they are already copied four times.

Reply:
Sure- will move it right after.